<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addToListButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>105</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>add to list</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>301</width>
      <height>268</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>mark of car:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="markCarEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>engine type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="EngineTypeEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>cost:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="costEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>country of origin:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="countryEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>fuel consumption per 100 km:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fuelEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>reliability:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="relabilityEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>comfort</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="comfortEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="view">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>641</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="correctButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>105</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>correct</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>429</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>delete car number â„–</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="deleteEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>20</y>
      <width>215</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>sort by mark:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortButton">
       <property name="text">
        <string>sort</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>70</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
